### ISSUE TITLE
There should be different pictures for each article

### ISSUE THREAD

**Issue by malagant** [RustMagazine/rustmagazine#49]
Looking at the current page all the images are the same.
Maybe we can get some other pictures from unsplash/gliphy or even use stable diffusion to generate AI driven images from the abstract of an article. What do you think?

**Comment by Folyd** [comment]
Yes, agree. Using AI such as stable diffusion to generate pictures is cool, especially, I hope the AI can generate a bunch of images similar to the style of the current cover picture.

This picture is from https://rustacean.net/, the origin post is https://twitter.com/_httprs/status/1394248700424302600.
![](https://rustmagazine.org/static/bg.jpeg) I love it so much, it feels relaxed, pleasant, and lovely. It gives the magazine current look and feels, and I also found the readers also love it.

Other pictures from unsplash/gliphy are also good choices. 👍

**Comment by malagant** [comment]
It all depends on the prompt. I can give it a try.

**Comment by pau1tuck** [comment]
I agree. It would be nice to have different banners of a similar style for each article.

**Comment by LCrossman** [comment]
I like the AI plan, but there are also good pictures on pixabay
